FHIR/src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/AnonymizerConfigurations/AnonymizationFhirPathRule.cs

```diff
@@ -6,19 +6,25 @@ namespace Microsoft.Health.Fhir.Anonymizer.Core.AnonymizerConfigurations
 {
     public class AnonymizationFhirPathRule : AnonymizerRule
     {
+        // Example: nodesByType('Patient').use
+        public static readonly Regex TypeRuleRegex = new Regex(@"^nodesByType\([\'|\""](?<type>[A-Za-z0-9]+)[\'|\""]\)(\.(?<expression>[A-Za-z0-9]+))?$");
```

Comment:
Do we support FHIRPath functions here?

Reply:
No, only children expression is supported. FHIRPath functions like `nodesByType("HumanName").nodesByType("Period")` will not work here.